tenant_schemas/tests/testcases.py

```diff
@@ -1,41 +1,37 @@
-import django
 import inspect
+
 from django.conf import settings
 from django.core.management import call_command
 from django.db import connection
-from django.test import TestCase
+from django.test import TransactionTestCase
 
 from tenant_schemas.utils import get_public_schema_name
 
 
-class BaseTestCase(TestCase):
+class BaseTestCase(TransactionTestCase):
     """
     Base test case that comes packed with overloaded INSTALLED_APPS,
     custom public tenant, and schemas cleanup on tearDown.
     """
     @classmethod
     def setUpClass(cls):
-        settings.TENANT_MODEL = 'tenant_schemas.Tenant'
-        settings.SHARED_APPS = ('tenant_schemas', )
+        settings.TENANT_MODEL = 'customers.Client'
+        settings.SHARED_APPS = ('tenant_schemas', 'customers')
         settings.TENANT_APPS = ('dts_test_app',
                                 'django.contrib.contenttypes',
                                 'django.contrib.auth', )
         settings.INSTALLED_APPS = settings.SHARED_APPS + settings.TENANT_APPS
 
-        # Django calls syncdb by default for the test database, but we want
-        # a blank public schema for this set of tests.
-        connection.set_schema_to_public()
-        cursor = connection.cursor()
-        cursor.execute('DROP SCHEMA %s CASCADE; CREATE SCHEMA %s;'
-                       % (get_public_schema_name(), get_public_schema_name(), ))
+        cls.available_apps = settings.INSTALLED_APPS
```

Comment:
What is this variable for? Please move to a separate PR if parallel migrations does not cause this test to fail.